src/Hl7.Fhir.Base/FhirPath/EvaluationContext.cs

```diff
@@ -29,6 +29,11 @@ public EvaluationContext(ITypedElement resource, ITypedElement rootResource)
             Resource = resource;
             RootResource = rootResource ?? resource;
         }
+        
+        public EvaluationContext(ITypedElement resource, ITypedElement rootResource, IDictionary<string, IEnumerable<ITypedElement>> environment) : this(resource, rootResource)
```

Comment:
Also update the documentation at https://docs.fire.ly/projects/Firely-NET-SDK/en/latest/fhirpath/evaluation-contexts.html about the new environment stuff.